tests/generate_fake_data.py

```diff
@@ -0,0 +1,39 @@
+from faker import Faker
```

Comment:
This was just used one time to create the testing database along with some manual spreadsheet processing.  I'd like to save the file, but it's not necessary for any build or testing.  I moved it to a util folder to avoid confusion.